skimage/restoration/_denoise.py

```diff
@@ -264,41 +264,53 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
 def denoise_tv_bregman(image, weight=5.0, max_num_iter=100, eps=1e-3,
                        isotropic=True, *, channel_axis=None,
                        multichannel=False):
-    """Perform total-variation denoising using split-Bregman optimization.
+    r"""Perform total variation denoising using split-Bregman optimization.
 
-    Total-variation denoising (also know as total-variation regularization)
-    tries to find an image with less total-variation under the constraint
-    of being similar to the input image, which is controlled by the
-    regularization parameter ([1]_, [2]_, [3]_, [4]_).
+    Given :math:`f` a noisy image (input data),
+    total variation denoising (also known as total variation regularization)
+    aims to find an image :math:`u` with less total variation than :math:`f`,
+    under the constraint that :math:`u` remain similar to :math:`f`.
+    This can be expressed by the Rudin--Osher--Fatemi (ROF) minimization
+    problem:
+
+    .. math::
+
+        \min_{u} \sum_{i=0}^{N-1} \left( \left| \nabla{u_i} \right| + \frac{\lambda}{2}(f_i - u_i)^2 \right)
+
+    where :math:`\lambda` is a positive parameter.
+    The first term of this cost function is the total variation;
+    the second term represents data fidelity. As :math:`\lambda \to 0`,
+    the total variation term dominates, forcing the solution to have smaller
+    total variation, at the expense of looking less like the input data.
+
+    This code is an implementation of the split Bregman algorithm of Goldstein
+    and Osher to solve the ROF problem ([1]_, [2]_, [3]_).
 
     Parameters
     ----------
     image : ndarray
-        Input data to be denoised (converted using img_as_float`).
-    weight : float
-        Denoising weight. The smaller the `weight`, the more denoising (at
-        the expense of less similarity to the `input`). The regularization
-        parameter `lambda` is chosen as `2 * weight`.
+        Input image to be denoised (converted using ``img_as_float``).
+    weight : float, optional
+        Denoising weight. It is equal to :math:`\frac{\lambda}{2}`. Therefore,
+        the smaller the `weight`, the more denoising (at
+        the expense of less similarity to `image`).
     eps : float, optional
-        Relative difference of the value of the cost function that determines
-        the stop criterion. The algorithm stops when::
-
-            SUM((u(n) - u(n-1))**2) < eps
-
+        Stop criterion. The algorithm stops when
+        :math:`\sqrt{(u_n - u_{n-1})^2} < eps`.
```

Comment:
You haven't missed any explanation, @lagru (and than you for your review, by the way). From the Goldstein and Osher reference, the lhs is the Euclidean norm of the difference between consecutive u's: This is what I intended to express with :math:`\sqrt{(u_n - u_{n-1})^2} < eps` but I was lazy and didn't include the sum over i's (vector components). There must be a normalization factor as well (I guess that what `total` is in the code): In a way, set "eps' = eps / N" and you're good (hehe)...

Reply:
@lagru how about 06b96ecd8ba766d2679701624eb90d40faa02a22?

Reply:
See https://github.com/scikit-image/scikit-image/pull/6544#discussion_r1004201524.

Otherwise, $||u_n - u_{n-1}||_2$ looks like a clear improvement over the previous two versions! Though, I'm still skeptical that this represents the nesting of successive sums and roots as the norm is applied on the current difference `tx` + the previous `rmse`. But I don't have a better suggestion. :wink: 

Btw, I find it curious that Sphinx seems to render `\| x \|` as $|| x ||$ while GitHub renders it to $\| x \|$.